src/sentry/static/sentry/app/views/releasesV2/detail/releaseHeader.tsx

```diff
@@ -26,11 +27,15 @@ type Props = {
   orgId: string;
   release: Release;
   deploys: Deploy[];
+  selection: GlobalSelection;
 };
 
-const ReleaseHeader = ({location, orgId, release, deploys}: Props) => {
+const ReleaseHeader = ({location, orgId, release, deploys, selection}: Props) => {
   const {version, newGroups, url} = release;
 
+  const healthData = release.projects.find(p => p.id === selection.projects[0])
```

Comment:
This whole implementation should have some kind of fallback as it can crash easily when no project is found in some odd scenario

Reply:
That shouldn't happen.

Reply:
I believe the worst-case scenario here is that the undefined will be assigned to healthData const.

Note the optional chaining wrapped to the second line.

We then handle undefined healthData down the road.